yt_dlp/extractor/tvw.py

```diff
@@ -115,3 +123,44 @@ def _real_extract(self, url):
                 'is_live': ('eventStatus', {lambda x: x == 'live'}),
             }),
         }
+
+
+class TvwNewsIE(TvwBaseIE):
+    IE_NAME = 'Tvw:News'
+    _VALID_URL = r'https?://(?:www\.)?tvw\.org/(\d{4})/(0[1-9]|1[0-2])/(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://tvw.org/2024/01/the-impact-issues-to-watch-in-the-2024-legislative-session/',
+        'info_dict': {
+            'id': 'the-impact-issues-to-watch-in-the-2024-legislative-session',
+            'title': 'The Impact - Issues to Watch in the 2024 Legislative Session',
+            'description': 'md5:65f0b33ec8f18ff1cd401c5547aa5441',
+        },
+        'playlist_count': 6,
+    }, {
+        'url': 'https://tvw.org/2024/06/the-impact-water-rights-and-the-skookumchuck-dam-debate/',
+        'info_dict': {
+            'id': 'the-impact-water-rights-and-the-skookumchuck-dam-debate',
+            'title': 'The Impact - Water Rights and the Skookumchuck Dam Debate',
+            'description': 'md5:185f3a2350ef81e3fa159ac3e040a94b',
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://tvw.org/2023/09/5th-annual-tvw-open-thank-you/',
+        'info_dict': {
+            'id': '5th-annual-tvw-open-thank-you',
+            'title': '5th Annual TVW Open THANK YOU!',
+            'description': 'md5:5306eef5b03c87108797cb6261c5f16c',
+        },
+        'playlist_count': 0,
```

Comment:
0? why?

Reply:
Because there are some news pages without any sort of video on them, so we want to make sure if there isn't any videos, it returns nothing.

Reply:
It can be removed